<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuadConfigDialog</class>
 <widget class="QDialog" name="QuadConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Quad Info</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="textureTypeComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Test Texture</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom Texture</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Video</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Solid Color</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>761</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="testTextureTab">
    <attribute name="title">
     <string>Test Texture</string>
    </attribute>
    <widget class="QLabel" name="testTextureDisplayLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>311</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="customTextureTab">
    <attribute name="title">
     <string>Custom Texture</string>
    </attribute>
    <widget class="QLabel" name="customTextureDisplayLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>310</y>
       <width>381</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="customTextureLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="customTextureOpenButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="videoTextureTab">
    <attribute name="title">
     <string>Video</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="solidColorTab">
    <attribute name="title">
     <string>Solid Color</string>
    </attribute>
    <widget class="QWidget" name="colorWidget" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="selectColorButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>250</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Color</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>490</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
